website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -108,3 +108,54 @@ To delete a published tag to re-tag, follow these steps:
     git push -d origin vx.y.z
 ```
 
+## Deploying RDS cluster and Cumulus ORCA modules in bamboo
+
+For testing, use the `feature/ORCA-test-bamboo` branch in cumulus-orca github repo and use the `ORCA-test-branch` linked repo in bamboo specs. 
+
+While running the `Deploy Dev RDS Stack` stage, replace the following variables with yours. This is because some variables are sensitive and some will vary depending upon the user running the pipeline. 
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
+- PREFIX
+- AWS_ACCOUNT_ID
+- DB_ADMIN_PASSWORD
+- DB_USER_PASSWORD

```

Comment:
Do DB_ADMIN_PASSWORD and DB_USER_PASSWORD need to be user-set? Seems like we could use a default value or, better yet, randomize them.

Reply:
since the variables have the word `password`, bamboo will not allow a default value other than an encrypted value.
So they have to be user-set during running the pipeline